code/getSampleRate.m

```diff
@@ -0,0 +1,31 @@
+function sratesout = getSampleRate(srates)
+%% input: matrix of sample rates of each packet from TimeDomainData.Json 
+%% output: sample rate in Hz 
+
+
+if length(unique(srates)) > 1 
+    warning('you have non uniform sample rates in your data'); 
+else
+    sratenum = unique(srates); 
+    switch sratenum 
+        case 0 
+            srate = 250; % sample rate in Hz. 
+        case 1 
+            srate = 500; 
+        case 2 
+            srate = 1e3; 
+    end
+end
+fprintf('out of %d packets:\n\t%d packets at 250Hz\n\t%d packets at 500Hz\n \t%d packets at 1000Hz\n',...
+    length(srates),...
+    sum((srates==0)),...
+    sum((srates==1)),...
+    sum((srates==2)));
+% make sample rates numbers 
+sratesout = srates; 
+sratesout(srates==0) = 250;
+sratesout(srates==1) = 500;
+sratesout(srates==2) = 1000;
+
+
+clear temp*; 
```

Comment:
Unnecessary use of clear since temp is not used in the function. If this assumes that a variables with temp in the name was created before, it still should not be cleared in this function, but in the file it was defined. This may break some scripts.

Reply:
Resolved with https://github.com/openmind-consortium/UCSF-rcs-data-analysis/pull/3/commits/3033bf9ad875f5e37de4b877c42e31f855cfdccc